src/bundle/Resources/config/services/normalizers.yaml

```
@@ -0,0 +1,30 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    Ibexa\Search\Serializer\SuggestionSerializerInterface:
+        class: Symfony\Component\Serializer\Serializer
+        autoconfigure: false
+        arguments:
+            $normalizers:
+                - '@Ibexa\Search\Serializer\Normalizer\Suggestion\ContentSuggestionNormalizer'
+                - '@Ibexa\Search\Serializer\Normalizer\Suggestion\ParentLocationNormalizer'
+            $encoders:
+                - '@serializer.encoder.json'
```

`Symfony\Component\Serializer\Serializer` is **not** an instance of `Ibexa\Search\Serializer\SuggestionSerializerInterface`.

This works because the moment `class` property is declared, the service name becomes just an ID.

I still think you should just create an instance of the serializer instead (without a dedicated interface) and use `container.service_subscriber` tag to indicate which serializer you want to inject.

```suggestion
    ibexa.search.serializer.suggestion_serializer:
        class: Symfony\Component\Serializer\Serializer
        autoconfigure: false
        arguments:
            $normalizers:
                - '@Ibexa\Search\Serializer\Normalizer\Suggestion\ContentSuggestionNormalizer'
                - '@Ibexa\Search\Serializer\Normalizer\Suggestion\ParentLocationNormalizer'
            $encoders:
                - '@serializer.encoder.json'
```